third_party/terraform/tests/resource_compute_instance_test.go

```diff
@@ -1784,6 +1784,42 @@ func TestAccComputeInstance_updateTerminated_desiredStatusRunning_notAllowStoppi
 	})
 }
 
+func TestAccComputeInstance_resourcePolicies(t *testing.T) {
```

Comment:
I'd mildly prefer this is a test on policy, so that if new fields are added we run it as a policy test.